contracts/MetaverseBaseNFT.sol

```diff
@@ -231,22 +228,26 @@ contract MetaverseBaseNFT is
     // function to disable gasless listings for security in case
     // opensea ever shuts down or is compromised
     // from CryptoCoven https://etherscan.io/address/0x5180db8f5c931aae63c74266b211f580155ecac8#code
-    function setIsOpenSeaProxyActive(bool _isOpenSeaProxyActive) public onlyOwner {
+    function setIsOpenSeaProxyActive(bool _isOpenSeaProxyActive)
+        public
+        onlyOwner
+    {
         isOpenSeaProxyActive = _isOpenSeaProxyActive;
     }
 
     // ---- Minting ----
 
-    function _mintConsecutive(uint256 nTokens, address to, bytes32 extraData) internal {
-        require(_tokenIdCounter.current() + nTokens + reserved <= maxSupply, "Not enough Tokens left.");
+    function _mintConsecutive(
+        uint256 nTokens,
+        address to,
+        bytes memory extraData
+    ) internal {
+        require(
+            totalSupply() + nTokens + reserved <= maxSupply,
```

Comment:
also! i wonder if some of the tokens were burned, should we allow to mint again?

probably we can use `totalMinted` instead of `totalSupply` here

Reply:
@caffeinum not really. inside of `totalSupply()` function is decreasing the burned token amount but `totalMinted()` function is not. `totalMinted()` function is just the number of total mints, not including burns.

Reply:
@caffeinum That makes sense. I pushed update for this.